src/server/app.py

```diff
@@ -0,0 +1,27 @@
+import os
+import pymongo
+from flask import Flask
+from pprint import pprint
+
+ADMIN_NAME = "SGP"
+PASSWORD = "UxmwrTNrtbmrHzGp"
```

Comment:
Don't commit this to github, otherwise random people will have access to the database because the repo is public. Either the repo will need to be made private or use environment variables instead.